vars/main.yml

```diff
@@ -18,14 +18,26 @@ jenkins_plugins:
   - ws-cleanup
   - xtrigger
   - envinject
-jenkins_scp_sites: []
+jenkins_scp_sites:
```

Comment:
Moved a bunch of variables that I had in my `cira_vars.yml` to a more public space. Can still be overridden, but now things should "just work" when you run a `vagrant up` or run against an OpenStack cloud (after you configure your authentication).